docs/connect/erlang.md

```diff
@@ -0,0 +1,84 @@
+(connect-erlang)=
+
+# Erlang
+
+:::{div} sd-text-muted
+Connect to CrateDB from Erlang applications.
+:::
+
+## ODBC
+
+:::{rubric} About
+:::
+
+Erlang includes an [ODBC application] out of the box that provides an
+interface to communicate with relational SQL-databases, see also
+[Erlang ODBC examples].
+
+:::{rubric} Synopsis
+:::
+
+`odbcinst.ini`
+```ini
+[PostgreSQL]
+Description     = PostgreSQL ODBC driver
+Driver          = /usr/local/lib/psqlodbcw.so
+```
```

Comment:
The `Driver` referencing `/usr/local/lib/psqlodbcw.so` is specific to macOS/Homebrew's `brew install psqlodbc`, and will be different on other OS.

Reply:
This has been improved with GH-411 per 48777c39.